méli-mélo/2021-05-steps/index.html

```diff
@@ -30,23 +30,32 @@ <h2>GCWeb implementation plan</h2>
 		 <h2>Classes used</h2>
 		 <ul class="colcount-sm-2 colcount-md-4">
 			<li><code>lst-stps</code></li>
+			 <li><code>ld-zr</code></li>
 			<li><code>stps-strpd</code></li>
 			<li><code>lst-stps-itm</code></li>
 			<li><code>lst-stps-sub</code></li>
 		 </ul>
-      <h2>Steps as an single-level <code>&lt;ol&gt;</code></h2>
+   <h2>Steps as an single-level <code>&lt;ol&gt;</code></h2>
   <div class="row">
     <div class="col-md-6">
       <h3>Default design</h3>
       <span class="wb-prettify all-pre"></span>
       <ol class="lst-stps">
-        <li>Lorem ipsum dolor sit amet, esse autem per ex, nec in magna lorem. Ea nec error scriptorem referrentur, patrioque torquatos conclusionemque pri in. Sed sint debitis ad, mazim vitae temporibus qui ut, vel erat ponderum efficiendi ei. In idque euismod sed.</li>
+        <li data-custom="blah">Lorem ipsum dolor sit amet, esse autem per ex, nec in magna lorem. Ea nec error scriptorem referrentur, patrioque torquatos conclusionemque pri in. Sed sint debitis ad, mazim vitae temporibus qui ut, vel erat ponderum efficiendi ei. In idque euismod sed.
+          <ul>
+            <li>Test</li>
+            <li>Test 2</li>
+          </ul>
+        </li>
         <li>List item 2</li>
         <li>
           <p>List item 3</p>
           <p>Second paragraph</p>
         </li>
-        <li>List item 4</li>
+        <li>
+          <h3>Header here</h3>
+          <p>Sed sint debitis ad, mazim vitae temporibus qui ut, vel erat ponderum efficiendi ei. In idque euismod sed.</p>
+        </li>
```

Comment:
Sectioning element are needed here.

```suggestion
        <li>
        <section>
          <h3>Header here</h3>
          <p>Sed sint debitis ad, mazim vitae temporibus qui ut, vel erat ponderum efficiendi ei. In idque euismod sed.</p>
          </section>
        </li>
```

There is a few instance like this through this working example page